articles/hypothesis/index.md

```diff
@@ -0,0 +1,112 @@
+
+### Contents
+
+1. Significance of hypotheses
+2. Hypothesis in ML vs statistics
+3. 5 steps to test a hypothesis
+
+### Prerequisites
+
+All that is required is a general understanding of concepts in [machine learning](education/supervised-learning-algorithms/) and statistics.
+
+### Significance of a Hypothesis
+
+A hypothesis can be described as a theory or argument which explains some observed phenomenon. In a scientific setting, a hypothesis is meant to be proven (or disproven) through experimentation.
+In data science, a crucial part of the modeling process is first coming up with a significant question or assumption. For example, we can say something like “different cars use the communal parking lot every day”. The framing of this statement makes it seem like we are declaring it to be true. However, using a statistical approach, it might be better to frame it as “the cars that use the parking lot are the same”. We have framed the assumption as a null hypothesis, which we shall seek to disprove. The first statement can be thought of as an alternative hypothesis. We shall define these two types of hypotheses later on.
+The significance of framing the two statements as we did above is to help eliminate randomness. We can compare it to the phrase “innocent until proven guilty” since we seek to disprove the null hypothesis and validate the alternative hypothesis. When these two hypotheses are tested, we seek to prove that the data we used is statistically significant. This means that occurrences were not by chance alone. We shall define statistical significance later in this article.
+
+### Statistical vs Machine Learning Hypotheses
+
+Even though most of the concepts we cover in this article are predominantly statistical, it is important to understand how the term hypothesis is perceived from either a purely statistical or machine learning perspective.
+
+When carrying out statistical hypothesis tests, we attempt to calculate the critical value, which shall be covered later on. We can refer to this critical value as an effect.
+The interpretation of the critical value is significant. It determines the likelihood of observing the effect if observations do not have a relationship. The suggestion that the effect is real comes about if the aforementioned likelihood is minute. If the likelihood is large, the effect is likely not to be real. In statistical hypothesis tests, there is no comment on the size of the effect. These tests are concerned with how likely the effect is present or absent in the population in consideration. This is based on the observed data samples.
+Statistical hypotheses are thus based on identifying the relationships between observations. They are probabilistic explanations about these relationships. Null and alternative hypotheses are denoted as H0 and Ha respectively.
```

Comment:
$H_0 and H_a$. I will make this change for now. However, note this change.

Reply:
Thank you for this. I'll be keener